README.md

```diff
@@ -1,128 +1,216 @@
 
-### Подготовка webdriver-сессии к работе
-Подготовка сессии к работе (например, установка специфических для пользователя команд) выполняется в секции `prepareBrowser`.
-Формат секции:
-```js
-prepareBrowser: function(browser) {
-    // do setup here
-}
-```
+Option name               | Description
+------------------------- | -------------
+`desiredCapabilities`     | **Required.** Used WebDriver [DesiredCapabilites](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)
+`sessionsPerBrowser`      | Number of sessions which are run simultaneously. Default value is `1`.
+`retry`                   | How many times test should be rerun. Default value is `0`.
 
-В данную функцию будет передана сессия `webdriver.io`.
+### grid
+Selenium grid URL. Default value is `http://localhost:4444/wd/hub`.
 
-### Кастомизация настройки окружения
+### baseUrl
+Base service url. Default value is `http://localhost`.
 
-Данные в конфиге можно изменять в зависимости от дополнительных условий в функции `prepareEnvironment`. Использование этой функции не обязательно, она для удобства.
+### timeout
+Timeeout for text execution. Default value is `60000` ms.
 
-## Плагины
-hermione поддерживает расширения.
-Плагин представляет из себя модуль, экспортирующий одну единственную функцию. Эта функция принимает на вход два параметра:
-* инстанс hermione
-* опции плагина из конфига
+### waitTimeout
+Timeout for web page events. Default value is `10000` ms.
 
-В момент запуска тестов плагин будет загружен и вызван с текущим инстансом hermione и опциями конфига, соответствующими данному плагину.
+### slow
+If test execution time is greater than this value, then test is slow. Default value is `10000`.
 
-Если название модуля плагина начинается с `hermione-`, то в конфиге этот префикс можно опустить. Если в зависимостях есть два модуля, например `hermione-some-module` и `some-module`, то приоритет отдается модулю с префиксом.
+### debug
+Turn webdriver debug mode on. Default value is `false`.
 
-Пример.
+### sessionsPerBrowser
+Number of sessions which are run simultaneously. Global value for all browsers. Default value is `1`.
+
+### retry
+How many times test should be retried in case of a fail. Global value for all browsers. Default value is `0`.
+
+### plugins
+`hermione` plugin provides an extra features extending built-in possibilities. For example, [hermione-allure-reporter](https://github.com/gemini-testing/hermione-allure-reporter) and [hermione-tunnel](https://github.com/gemini-testing/hermione-tunnel).
+
+Plugin is a module which exports a single function. The function has two arguments:
+* hermione instance;
+* plugin options from configuration file.
+
+Plugins will be loaded before `hermione` runs tests.
+
+If a plugin name starts with `hermione-`, then the prefix can be ommited in the configuration file. If two modules with names `hermione-some-module` and `some-module` are specified, then module with prefix will have higher priority.
```

Comment:
I'll add the suggestion about naming and remove this sentence about module priority.